packer/scripts/install-dnsmasq.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+echo "Installing dnsmasq..."
+sudo apt-get update update -y
```

Comment:
weird double update :thinking: but the other script did this as well

Reply:
Let's just remove it